frontend/src/components/layout/SideNavBar/__snapshots__/SideNavbar.test.tsx.snap

```diff
@@ -134,7 +134,7 @@ exports[`SideNavbar display and logic renders 1`] = `
   width: 100%;
 }
 
-@media (max-width:76.5rem) {
+@media (max-width:1225px) {
```

Comment:
media queries are locked to 16px->1rem, so easier to just leave them as px to avoid confusion.